src/Action/PlaceholderAction.php

```diff
@@ -25,7 +25,7 @@ final class PlaceholderAction
      *
      * @return object
      */
-    public function __invoke($data)
+    public function __invoke($data = null)
```

Comment:
Should I maybe do a new controller instead of changing this?

Reply:
Yes, a new one that directly throw a 404 (so we can simplify the metadata).

Reply:
I thought we need to do it in the item data provider?